Array/SortArrayByParity.swift

```diff
@@ -8,11 +8,9 @@
 
 class SortArrayByParity {
     func sortArrayByParity(_ A: [Int]) -> [Int] {
-        var outputArray = [Int]()
-        for (_,value) in A.enumerated(){
-            outputArray.insert(value, at: value % 2 == 0 ? 0 : outputArray.count)
+        return A.enumerated().reduce(into: [Int]()) { (acc, arg) in
```

Comment:
I guess it's a question for the old change: why do we need to call `enumerated()` here?

Reply:
Okay, There are ways. But It's all about, how we code 🗡

Reply:
I am ok to remove `enumerated()`.